webrender/src/device/mod.rs

```diff
@@ -0,0 +1,385 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use api::{DeviceUintSize, ImageFormat, TextureTarget};
+#[cfg(not(feature = "gfx"))]
+use gleam::gl::{GLuint, DYNAMIC_DRAW, STATIC_DRAW, STREAM_DRAW};
```

Comment:
let's reverse this: have `gleam` as an optional dependency/feature to enable the old backend, and use `gfx` otherwise